src/components/AttentionBox/AttentionBox.jsx

```diff
@@ -7,6 +7,7 @@ import AlertIcon from "../Icon/Icons/components/Alert";
 import { baseClassName, closeClassName, compactClassName, ATTENTION_BOX_TYPES } from "./AttentionBoxConstants";
 import "./AttentionBox.scss";
 import { backwardCompatibilityForProperties } from "../../helpers/backwardCompatibilityForProperties";
+import { useA11yNotification } from "../../hooks/useA11yNotification";
```

Comment:
**Please not merge this PR until I return from my vacation (I want to check it in dapulse right after release the new version)**